editor.planx.uk/src/@planx/components/Pay/model.ts

```diff
@@ -1,9 +1,14 @@
 import { useStore } from "pages/FlowEditor/lib/store";
 import { ApplicationPath } from "types";
-import { boolean, object, string } from "yup";
+import { array, boolean, object, string } from "yup";
 
 import type { MoreInformation } from "../shared";
 
+export interface GovPayMetadata {
```

Comment:
Spent a bit of time trying to get the right "shape" for this. 

Settled on this as it makes form validation and population easy. Validating dynamic key values proved more tricky than it was worth. It just means there will be a step of formatting this for GovPay on the API side of things.